cmake/IgnConfigureProject.cmake

```diff
@@ -56,8 +56,14 @@ macro(ign_configure_project)
   # Extract the designation
   #============================================================================
   set(IGN_DESIGNATION ${PROJECT_NAME})
-  # Remove the leading "ignition-"
-  string(REGEX REPLACE "ignition-" "" IGN_DESIGNATION ${IGN_DESIGNATION})
+  # Remove the leading project prefix ("ignition-" by default)
+  set(PROJECT_PREFIX "ignition")
```

Comment:
Can we default to `gz` and add a note to the block below to remove `ignition-` after all libraries have migrated?

Reply:
https://github.com/ignitionrobotics/ign-cmake/pull/246/commits/536ea63ce3c0ad6d2b3f4aa812acaefb7df67e50